src/validations/project.validation.js

```diff
@@ -28,8 +28,30 @@ const getProject = {
   })
 };
 
+const getAllMessages = {
+  params: Joi.object().keys({
+    projectId: Joi.string().custom(objectId),
+    classId: Joi.string().custom(objectId)
+  })
+};
+
+const addMessage = {
+  params: Joi.object().keys({
+    projectId: Joi.string().custom(objectId),
+    classId: Joi.string().custom(objectId)
+  }),
+  body: Joi.object().keys({
+    message: Joi.string().required(),
+    from: Joi.string().custom(objectId),
+    to: Joi.string().custom(objectId),
+    classId: Joi.string().custom(objectId)
```

Comment:
why is this needed in the body?

Reply:
you mean classId?

Reply:
we need 'message' and 'to', and as you said 'from' is unnecessary. we leave it like this until frontend team have other requirements. 'classId' is a mistake, i delete it

Reply:
i have no permission to directly work on master branch, could you delete it for me ?